docs/topics/api/auth.rst

```diff
@@ -4,8 +4,8 @@
 Authentication (External)
 =========================
 
-To access the API as an external consumer, you need to include a
```

Comment:
That might not be enough... Maybe we should have a paragraph like `Each endpoint should specify whether it needs or would benefit from authenticated calls in some way.` before that ?

Reply:
Yeah, add a follow-on sentence saying something like that.  Suggestion: `See the documentation for individual endpoints where it will indicate if that endpoint requires, or would benefit from, authentication.`

Reply:
It felt a little awkward after the sentence, so added that bit inside parenthesis. Let me know what you think.